### ISSUE TITLE
OpenAPI specifiocation download not working

### ISSUE THREAD

**Issue by kine** [MicrosoftDocs/navdevitpro-content-pr#2139]
The link to download OpenAPI specification for API v2 is not working. Returning Error 404.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f73b29a3-942d-079d-08bf-4ab67ec213ee
* Version Independent ID: f236c724-5fb1-dbc2-b12e-ceb49e901af7
* Content: [OpenAPI Specification - Dynamics NAV](https://docs.microsoft.com/en-us/dynamics-nav/api-reference/v2.0/dynamics-open-api)
* Content Source: [dynamics-nav/api-reference/v2.0/dynamics-open-api.md](https://github.com/MicrosoftDocs/navdevitpro-content-pr/blob/live/dynamics-nav/api-reference/v2.0/dynamics-open-api.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by edupont04** [comment]
Hello Kamil,

I suspect a temporary glitch since the link works just fine for me just now. Can you try again?

**Comment by kine** [comment]
Hi Eva,
no, still Error 404. The download leads to https://docs.microsoft.com/en-us/dynamics-nav/api-reference/v2.0/contracts/BCOAS1.0.yaml

Another thing is that it is page for API v2 but in the text is the link marked as v1.0. File name is 1.0 too. Isn't it some mixture of versions?

Because https://docs.microsoft.com/en-us/dynamics-nav/api-reference/v1.0/contracts/BCOAS1.0.yaml works and it give me v1.0. I have tried even https://docs.microsoft.com/en-us/dynamics-nav/api-reference/v2.0/contracts/BCOAS2.0.yaml but not exists too.

**Comment by edupont04** [comment]
Yes, you're right - there isn't a YAML file for v2.0. I don't know why, and the people who do are on vacation this week (school break here in Denmark). Until they are back, I'll update the link to make it clear that it's for v1.0.

**Comment by henrikwh** [comment]
Hi @kine 
We dont have an OpenAPI spec for V2.0 of the APIs. I suggest that you try an use this project from GitHub to generate OpenAPI spec from OData metadata.
https://github.com/microsoft/OpenAPI.NET.OData

That should get you started on an OpenAPI spec.

**Comment by BortellZ** [comment]
Hello,

The V1 specification link now seems to go to a 404.

https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/api-reference/v1.0/contracts/BCOAS1.0.yaml

There is also confusingly a page for the V2 specifications, but no actual download link to it. The page also makes reference to the non-existent link.

https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/api-reference/v2.0/dynamics-open-api
